src/types/Select.ts

```diff
@@ -63,6 +63,7 @@ interface ISelectSingle<T> extends IBaseSelect<T> {
   multiple: false;
   onChange: (value: SingleSelectDataType<T>) => void;
   value?: SingleSelectDataType<T>;
+  optionsNumber?: number;
```

Comment:
The `optionsNumber` property can be put into `IBaseSelect` so both Single and Multiple will inherit